dsc_lib/src/configure/mod.rs

```diff
@@ -185,6 +190,18 @@ fn add_metadata(kind: &Kind, mut properties: Option<Map<String, Value>> ) -> Res
         return Ok(serde_json::to_string(&properties)?);
     }
 
+    if let Some(resource_metadata) = resource_metadata {
+        let other_metadata = resource_metadata.other;
+        if let Some(mut properties) = properties {
+            properties.insert("_metadata".to_string(), Value::Object(other_metadata));
```

Comment:
I think we need to check if the resource has a `_metadata` property before we add it otherwise it will invalidate their schema.  There should be a test added for this as well.  In the case the resource doesn't handle `_metadata`, then we don't send it.